### ISSUE TITLE
Feature request: Add sortby TV to available options

### ISSUE THREAD

**Issue by pepebe** [modxcms/Archivist#1]
Today I stumbled upon an old forum post by tsnekifox: http://forums.modx.com/thread/81834/sortbytv-for-archivist

> Would adding the ability to sort by a date TV be something easily done for Archivist? ... in cases where you want to do an archive for an event list instead of a blog I think it would be helpful to be able to sort by a TV for the event date as apposed to the publish date.

I think this is a very good idea and it should be added to a future version of archivist.

Regards,

pepebe

P.S. I posted a dirty workaround to this problem in the thread. Anyone with a similar problem should have a look for it.

**Comment by 4ntz** [comment]
+1 for that feature. I do exactly need this and playing around with &sortBy did not lead to success with my custom date-TV. Am I doing something wrong or is this still missing?

I noticed that you can set the publishedon date to a date in the future which results in being processed the way as you would expect it to (if i set there a date now in the year 2015 i get a new category listed)... but i would rather prefer a custom date-TV for that purpose.

**Comment by shaunmorrison** [comment]
Agreed - this is much needed. If someone un-publishes and re-publishes  a document, by default the publishedon date changes to the current date/time. By using a TV you can eliminate this issue.

**Comment by navtsi** [comment]
+1. Need it for a very simple event listing.
